stories/index.stories.js

```diff
@@ -238,40 +239,21 @@ storiesOf('Modal', module)
 
 storiesOf('Navbar', module)
   .add('Default', () => (
-    <Navbar expand="lg" variant="">
-      <Navbar.Brand href="#">{'MusicBrainz'}</Navbar.Brand>
-      <Navbar.Toggle aria-controls="mb-navbar-nav" />
-      <Navbar.Collapse id="mb-navbar-nav">
-        <Nav className="mr-auto">
-          <Nav.Link href="#login" onClick={noop}>{'Login'}</Nav.Link>
-          <Nav.Link href="#register" onClick={noop}>{'Register'}</Nav.Link>
-          <NavDropdown id="mb-nav-dropdown" title="About Us">
-            <NavDropdown.Item href="#about" onClick={noop}>
-              {'About MusicBrainz'}
-            </NavDropdown.Item>
-            <NavDropdown.Item href="#sponsors" onClick={noop}>
-              {'Sponsors'}
-            </NavDropdown.Item>
-            <NavDropdown.Item href="#team" onClick={noop}>
-              {'Team'}
-            </NavDropdown.Item>
-            <NavDropdown.Divider />
-            <NavDropdown.Item href="#licenses" onClick={noop}>
-              {'Data Licenses'}
-            </NavDropdown.Item>
-          </NavDropdown>
-        </Nav>
-        <Form inline>
-          <FormControl placeholder="Search" type="text" />
-          <FormControl as="select" className="custom-select">
-            <option>{'Artist'}</option>
-            <option>{'Recording'}</option>
-            <option>{'Work'}</option>
-          </FormControl>
-          <Button variant="outline-success">{'Search'}</Button>
-        </Form>
-      </Navbar.Collapse>
-    </Navbar>
+    <Header
+      projectName="musicbrainz"
+      searchOptions={searchOptions}
+    />
+  ));
+
+storiesOf('Footer', module)
+  .add('Default', () => (
+    <Footer
```

Comment:
Looking to do this way for all the projects and reduce boilerplate code. @mwiencek @MonkeyDo

Reply:
We can't really include components in this repo that use translated strings, since we have no way of extracting them here (each project will have different i18n setups).  At best we could pass the strings in as props, but that would probably add more boilerplate than you're saving.